### DISCUSSION TITLE
崩铁获取跃迁记录的api还是用不了嘛？auth key b 长度不对

### DISCUSSION THREAD

**Discussion by a361570441** [UIGF-org/mihoyo-api-collect#61] [Q&A]

**Comment by MoRanYue** [comment]
emm，星穹铁道早就可以在米游社的“联系客服”页面查找到跃迁记录，但这里暂时并没有深入研究

**Comment by a361570441** [comment]
> emm，星穹铁道早就可以在米游社的“联系客服”页面查找到跃迁记录，但这里暂时并没有深入研究

您好！感谢您的回答 但是我在联系客服页面只能找到初始池子的跃迁记录，请问是我的查找方式不对吗

**Comment by MoRanYue** [comment]
![Screenshot_2023-12-20-07-02-57-885_com.guojian.yuluo-edit.png](https://github.com/UIGF-org/mihoyo-api-collect/assets/84905594/2f89d484-800b-4f8c-b541-0dc455563c91)

确实只有始发跃迁呀

**Comment by frankcwl** [comment]
所以现在有不通过客户端获取抽卡链接或者authkey的方法吗

**Comment by MoRanYue** [comment]
> 所以现在有不通过客户端获取抽卡链接或者authkey的方法吗

这里不知道呀，似乎没有被找到呢

**Comment by Vocaloid2048** [comment]
> > 所以现在有不通过客户端获取抽卡链接或者authkey的方法吗
> 
> 这里不知道呀，似乎没有被找到呢

想請問一下 目前有找到解嗎？感謝

**Comment by ZianTT** [comment]
抓了下包，米游社里面authkeyb也是用的现有的方法，和游戏的authkey也对不上。这个authkey也不能复用，同样客服中心也没开放除了始发以外的api，希望有大佬抓一下客户端看看authkey从哪来的（https://github.com/UIGF-org/mihoyo-api-collect/issues/40#issuecomment-1983776127）

**Comment by MoRanYue** [comment]
> 抓了下包，米游社里面authkeyb也是用的现有的方法，和游戏的authkey也对不上。这个authkey也不能复用，同样客服中心也没开放除了始发以外的api，希望有大佬抓一下客户端看看authkey从哪来的（https://github.com/UIGF-org/mihoyo-api-collect/issues/40#issuecomment-1983776127）

（emm，这里认为，客户端中，应该是TCP包，而且，是加密的ProtoBuffer内容呀
或许，获取它的逻辑，或许，并不特别轻松呢）

**Comment by ZianTT** [comment]
> （emm，这里认为，客户端中，应该是TCP包，而且，是加密的ProtoBuffer内容呀
> 或许，获取它的逻辑，或许，并不特别轻松呢）

看来星铁自动获取跃迁数据遥遥无期啊（悲）

**Comment by ChenVAV** [comment]
在使用网页版的云崩铁进行分析时，我发现 authkey 可能是通过 WebSocket 发送特定数据帧，并从服务端返回的二进制数据中解析获得的，服务端发送和接收的数据都是二进制格式。感觉要拿到非常麻烦QAQ
![9ADBCE52-35A6-4790-A046-DB666A22DEB8](https://github.com/user-attachments/assets/6ca2bbd7-fd06-48d8-bb00-1a39d9cbf05d)

**Reply by iceprosurface** [reply]
是从 sr.mihoyo.com/cloud/#/ 抓的么？
